app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_many :subsciption_ids, class_name: 'UserSubscription', dependent: :destroy, counter_cache: :subscriptions_count, foreign_key: :user_id, inverse_of: :user 
```

Comment:
let's simply name this as user_subscriptions so we don't have to pass in so many options